test/components/common/stat/Stat.test.js

```diff
@@ -308,6 +308,7 @@ describe('Stat', () => {
       });
 
       it('should render the summary data when `isOpened` state is `false`', () => {
+        wrapper.setState({ isOpened: true });
```

Comment:
As discussed, I was expecting to see these doubles throughout the tests.  I'm guessing you tried it, but in case not have you tried running `wrapper.update()` instead of the 2nd `setState` call?  (I'd test locally myself, but there's currently an npm registry issue preventing me from building)

Reply:
Yeah, unfortunately `wrapper.update()` didn't do anything. Having a breakpoint just after the first `setState` shows that it, indeed, doesn't actually set the state while the second round does. I'm afraid that, barring digging into `enzyme` code directly, it's going to be difficult to pin the cause of that down.